source/_integrations/onedrive.markdown

```diff
@@ -32,9 +32,17 @@ Client ID:
   description: "Application ID of the app registration to be used with the integration. Uses Home Assistant provided by default."
 Client secret:
   description: "Application secret for the app registration. Uses Home Assistant provided by default."
+Folder name:
+  description: "The name of the instance specific [backup folder](#backup-folder)."
 
 {% endconfiguration_basic %}
 
+## Backup folder
+
+The backup folder is `root:\Apps\[Home Assistant | Graph]\backups_{id}`. This is not configurable, because otherwise the integration would need permissions to write in your entire drive. You can however rename the application folder which is called `Home Assistant` or `Graph` in your OneDrive. 
```

Comment:
```suggestion
The backup folder is `root:\Apps\[Home Assistant | Graph]\backups_{id}`. This is not configurable because otherwise the integration would need permissions to write into your entire drive. You can, however, rename the application folder which is called `Home Assistant` or `Graph` in your OneDrive. 
```